src/Services/ValidationService.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace ITBugTracking\Services;
+
+use ITBugTracking\Hydrators\SeverityHydrator;
+use PDO;
+
+class ValidationService
+{
+
+    public static function checkRequiredDataExists($data): bool
+    {
+        return !empty($data['reporter']) || !empty($data['title']) || !empty($data['severity']) || !empty($data['department']);
+    }
+
+
+    //sanitize 255 character limits
+    public static function limitTitleCharLengthTo255($data)
+    {
+        if (isset($data)) {
+
+            if (strlen($data) > 255) {
+                $data = substr($data, 0, 255);
+            }
+            return $data;
+        }
+    }
+
+    public static function limitReporterCharLengthTo255($data)
+    {
+        if (strlen($data) > 255) {
+            $data = substr($data, 0, 255);
+        }
```

Comment:
```suggestion
        return substr($data, 0, 255);
```

You don't need to test the length of the string first, you can just return this substr. 

[In the manual](https://www.php.net/manual/en/function.substr.php) you can see that if the string is shorter than the substring it'll just return the same string - 'the string returned will contain at most length characters'.

Reply:
done